source/manual/get-started.html.md

```diff
@@ -88,7 +80,19 @@ If you have a YubiKey, you will use `gpg-agent` in place of `ssh-agent`, which r
 [create-gpg-key]: /manual/create-a-gpg-key.html
 [generate-ssh-key]: https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
 
-## 3. Set up your GitHub account
+## 3. Set up your Slack profile
+
+As part of your onboarding to GDS you will have been given access to Slack - if you've not got this please talk to your tech lead.
+
+You should take a moment to configure your Slack profile so that others on Slack are able to determine who you are quickly.
+
+Please update the title field to include:
+
+- your role
+- which team you're working on
+- if you've joined through a supplier, their name.
```

Comment:
Nit: if you wanted you could make this more concise:

```suggestion
Help others know who you are by updating your Slack profile's 'title' field. This should include:

- your job role
- the team you're working on
- if relevant, the name of your supplier
```

Reply:
Could also add a link to slack's docs about where to find your profile: https://slack.com/intl/en-gb/help/articles/204092246-Edit-your-profile